soc/nxp/kinetis/ke1xz/Kconfig

```diff
@@ -12,3 +12,14 @@ config SOC_SERIES_KE1XZ
 	select CLOCK_CONTROL
 	select HAS_MCUX
 	select PLATFORM_SPECIFIC_INIT
+
+if SOC_MKE17Z9
+
+config SEL_LPO_1KHZ_CLOCK
```

Comment:
move this option to `drivers/counter/Kconfig.mcux_rtc` so other SoCs can benefit from it. Make this new option `depends on` the SoC that have it available. Please add `SOC_SERIES_S32K1` as a dependency as well.

Reply:
Ok, will do

Reply:
Done